### ISSUE TITLE
getPushReaction mixin fails

### ISSUE THREAD

**Issue by deadspidervenom** [williambl/piston-control-forge#1]
As you asked on here https://github.com/williambl/piston-control/issues/2
These are my last log and and relevant crash report before i removed your mod (the reason i say relevant was i removed a batch of mods till it worked and had a few client side only mods crash it or forgot core files, when i started adding mods back in)

https://pastebin.com/7reAc1K7

https://pastebin.com/t0jZpnjH

I would like to note a few things.

1. I honestly lost track of which reports and logs were for what, ill try to get some new ones in a few hours after i sleep, just tired rn and spent a log time solving server errors4. So these could be unrelated if so i apologize.
2. The error i kept getting to my limited log knowledge kept pointing to unrelated mods, as it pointed at ones i knew ran but when removed would just point at another.
3. I know it has to do with piston control as removing it made the server load, what exactly idk, heck it could be another mod entirely that just does not like piston controls code, i just am too inexperienced to know.

**Comment by deadspidervenom** [comment]
Apologies for the wait, this is a for sure related log

https://pastebin.com/qzFjZPGr

There was no crash report

**Comment by williambl** [comment]
Yeah, that's the log I need. Thanks. I'll take a look at why that's happening.

**Comment by jethroucc** [comment]
any progress?  no other piston control mods for 1.15.2!  IF you need any help testing or anything, i'd be happy to do so.
